packages/editor/package.json

```diff
@@ -0,0 +1,71 @@
+{
+	"name": "@wordpress/editor",
+	"version": "1.0.0-alpha.1",
+	"description": "Building blocks for WordPress editors",
+	"author": "The WordPress Contributors",
+	"license": "GPL-2.0-or-later",
+	"keywords": [
+		"wordpress",
+		"editor"
+	],
+	"homepage": "https://github.com/WordPress/gutenberg/tree/master/packages/editor/README.md",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/WordPress/gutenberg.git"
+	},
+	"bugs": {
+		"url": "https://github.com/WordPress/gutenberg/issues"
+	},
+	"main": "build/index.js",
+	"module": "build-module/index.js",
+	"react-native": "src/index",
+	"dependencies": {
+		"@babel/runtime": "^7.0.0-beta.52",
+		"@wordpress/a11y": "file:../a11y",
+		"@wordpress/api-fetch": "file:../api-fetch",
+		"@wordpress/blob": "file:../blob",
+		"@wordpress/blocks": "file:../blocks",
+		"@wordpress/components": "file:../components",
+		"@wordpress/compose": "file:../compose",
+		"@wordpress/core-data": "file:../core-data",
+		"@wordpress/data": "file:../data",
+		"@wordpress/date": "file:../date",
+		"@wordpress/deprecated": "file:../deprecated",
+		"@wordpress/dom": "file:../dom",
+		"@wordpress/element": "file:../element",
+		"@wordpress/hooks": "file:../hooks",
+		"@wordpress/html-entities": "file:../html-entities",
+		"@wordpress/i18n": "file:../i18n",
+		"@wordpress/is-shallow-equal": "file:../is-shallow-equal",
+		"@wordpress/keycodes": "file:../keycodes",
+		"@wordpress/nux": "file:../nux",
+		"@wordpress/url": "file:../url",
+		"@wordpress/viewport": "file:../viewport",
+		"@wordpress/wordcount": "file:../wordcount",
+		"classnames": "^2.2.5",
+		"dom-react": "^2.2.1",
+		"dom-scroll-into-view": "^1.2.1",
+		"element-closest": "^2.0.2",
+		"lodash": "^4.17.10",
+		"memize": "^1.0.5",
+		"querystring": "^0.2.0",
+		"querystringify": "^1.0.0",
```

Comment:
We use multiple `querystring` libraries, we should settle on one. I'm leaving this for a later PR though.

Reply:
And Gutenberg itself also requires [`http-build-query`](https://www.npmjs.com/package/http-build-query) to build PHP-compatible query strings (because PHP has to be special), which adds to the query string fun :)